apps/omg_watcher/lib/omg_watcher/api/in_flight_exit.ex

```diff
@@ -106,7 +105,7 @@ defmodule OMG.Watcher.API.InFlightExit do
           {<<>>, <<>>}
 
         utxo_pos ->
-          with {:ok, %{proof: proof, txbytes: txbytes}} <- DB.TxOutput.compose_utxo_exit(utxo_pos),
+          with {:ok, %{proof: proof, txbytes: txbytes}} <- OMG.Watcher.API.Utxo.compose_utxo_exit(utxo_pos),
```

Comment:
Please alias OMG.Watcher.API for better visibility (probably 1st place API reuse other API module)
```suggestion
          with {:ok, %{proof: proof, txbytes: txbytes}} <- API.Utxo.compose_utxo_exit(utxo_pos),
```